src/ImageSharp.Formats.Png/PngEncoderCore.cs

```diff
@@ -495,7 +495,7 @@ private QuantizedImage<TColor> WritePaletteChunk<TColor>(Stream stream, PngHeade
 
             if (this.Quantizer == null)
             {
-                this.Quantizer = new WuQuantizer<TColor>();
+                this.Quantizer = new OctreeQuantizer<TColor>();
```

Comment:
You mean the PNG Encoder ? 

The `WuQuantizer` doesn't support dithering (for now) as I couldn't get the output correct so I switched it for the octree one which does. If you're encoding indexed png's you really want the output to be top notch so I thought the switch was important.